app/src/main/java/com/josycom/mayorjay/flowoverstack/util/AppConstants.java

```diff
@@ -14,7 +14,7 @@
 
     String QUESTION_FILTER = "!9Z(-wwYGT";
     String ANSWER_FILTER = "!9Z(-wzu0T";
-    String API_KEY = "YOUR_API_KEY";
+    String API_KEY = "QUEuzCqtqLg4NXnEOO5qug((";
```

Comment:
Please remove this key

Reply:
I'll fix it all. Thanks for pointing it out